appinfo/info.xml

```diff
@@ -31,6 +31,7 @@ This extension is the successor of the [ownCloud Objectstore App](https://market
 	</types>
 	<commands>
 		<command>OCA\Files_Primary_S3\Command\ls</command>
+		<command>OCA\Files_Primary_S3\Command\createBucket</command>
```

Comment:
why lowercase?!? It is a gooddamn class

Reply:
this was asked and responded to in the other pr already.

the app does not yet follow the OCA namespace semantics - actually the objectstore loading code in core is not respecting it. This needs a debugging session of it's own